runtime/src/alpha.rs

```diff
@@ -223,6 +300,27 @@ where
             }
         };
 
+        *self.state.lock().unwrap() = State::Loading;
+
+        let state = self.state.clone();
+        let cancellation_token = self.cancellation_token.clone();
+
+        // State and cancellation is not used in alpha
+        if self.beta {
+            // Ensure that the runtime is set to unhealthy if it doesn't reach the running state after
+            // it has sent a load response, so that the ECS task will fail.
+            tokio::spawn(async move {
+                // Note: The timeout is quite low as we are not actually provisioning resources after
+                // sending the load response.
```

Comment:
@oddgrd this isn't the case anymore now, is it ? Should we increase the timeout or do something else ?

Reply:
We have a 120 second timeout in the provisioner rds sdk, so I think we can keep it like this for now, and we'll see if we need to change it as I test the RDS resource implementation.

Reply:
We can update the comment, though, to give the reader an understanding of why we need it, which is the fact that some resources can take a while to provision, and at the time of writing the main offender is RDS.